### PR TITLE
Look what I did

### PR DESCRIPTION
**Pull request by robertcoopercode**
See Netlify deploy preview for a demo.

### CODE REVIEW

File: src/components/ThemeSwitcher.jsx [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import React, { Component } from 'react';
-import { Toggle } from './Toogle';
+import { Toggle } from './Toggle';
```
Comment (benoitdeziel) [inline src/components/ThemeSwitcher.jsx]: 😂

Comment (robertcoopercode) [reply src/components/ThemeSwitcher.jsx]: We can't be having typos like this man.

Comment (benoitdeziel) [reply src/components/ThemeSwitcher.jsx]: soory

Comment (robertcoopercode) [reply src/components/ThemeSwitcher.jsx]: #triggered

**Comment by benoitdeziel** [comment]
Can you add a variation `danger` for the `ThemedButton` and one button disabled

**Comment by robertcoopercode** [comment]
You'd like a `danger` variation of the Custom Themed Button? You'd also want the `disabled` state for both the regular Custom Themed Button as well as the `danger` variation?

**Comment by benoitdeziel** [comment]
I will update the master branch with:

- Default Button
- Danger Button
- Disabled Button
- Disabled Danger Button
- Themed Button
- Themed Danger Button
- Disabled Themed Button
- Disabled Themed Danger Button

**Comment by benoitdeziel** [comment]
Can you try to resolve the conflicts, I want to check if it deploys on netlify

**Comment by robertcoopercode** [comment]
Anything for you

**Comment by robertcoopercode** [comment]
Yup, it works 👍

**Pull request closed without merging**